### PR TITLE
l3cam_ros: 0.0.3-2 in 'noetic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by adrisubi**
Increasing version of package(s) in repository `l3cam_ros` to `0.0.3-2`:

- upstream repository: https://github.com/beamaginelidar/l3cam_ros.git
- release repository: https://github.com/beamaginelidar/l3cam_ros-release.git
- distro file: `noetic/distribution.yaml`
- bloom version: `0.11.2`
- previous version for package: `null`

## l3cam_ros

```
* Added allied Wide and Narrow cameras
* Implemented default parameters
* Bug fixes
* Disable dynamic network configuration
* Contributors: Adrià Subirana
```

### CODE REVIEW

File: noetic/distribution.yaml [hunk 1]
```diff
@@ -4409,6 +4409,22 @@ repositories:
...
     status: maintained
+  l3cam_ros:
+    doc:
+      type: git
+      url: https://github.com/beamaginelidar/l3cam_ros.git
+      version: master
+    release:
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/beamaginelidar/l3cam_ros-release.git
```
Comment (quarkytale) [inline noetic/distribution.yaml:4420]: Seems to be missing, got a 404 on this url

**Review by quarkytale — DISMISSED** [review]

File: noetic/distribution.yaml (hunk 1 above)
Comment (adrisubi) [reply noetic/distribution.yaml:4420]: When copy-pasting the url into mozilla firefox it is found. I don't really understand why it got 404.
Screenshot 2023-03-28 09:58:20

Comment (nuclearsandwich) [reply noetic/distribution.yaml:4420]: @adrisubi by that screenshot it appears that this release repository is private. The release repository must be publicly visible in order for our builds to use it.

Comment (adrisubi) [reply noetic/distribution.yaml:4420]: Oh my bad, didn't notice it was private, now it's public. Tank you

**Review by nuclearsandwich — APPROVED** [review]

**Comment by nuclearsandwich** [comment]
CI was still unhappy. I merged this branch with master to see if it was a synchronization issue or confirm that there's something wrong with the change itself.

**Comment by lopsided98** [comment]
The release repository for this package appears to have been deleted.

**Pull request merged**